test/pylib/rest_client.py

```diff
@@ -0,0 +1,80 @@
+#
+# Copyright (C) 2022-present ScyllaDB
+#
+# SPDX-License-Identifier: AGPL-3.0-or-later
+#
+"""Asynchronous helper for Scylla REST API operations.
+"""
+import os.path
+import aiohttp
+
+
+class UnixRESTClient():
+    """An async helper for REST API operations using AF_UNIX socket"""
+    session: aiohttp.ClientSession
+
+    def __init__(self, sock_path: str):
+        self.sock_name = os.path.basename(sock_path)
+        self.session = aiohttp.ClientSession(connector = aiohttp.UnixConnector(path=sock_path))
+
+    async def stop(self) -> None:
+        """End session"""
+        await self.session.close()
+
+    def _resource_uri(self, resource: str) -> str:
+        # NOTE: using Python requests style URI for Unix domain sockets to avoid using "localhost"
+        return f"http+unix://{self.sock_name}{resource}"
+
+    async def get(self, resource: str) -> aiohttp.ClientResponse:
```

Comment:
Please think of a way to reduce the copy pasta.
Could the bodies of `get`, `get_text` and `put_json` be refactored to reusable functions?

Reply:
Common base class with these methods implemented in https://github.com/scylladb/scylladb/commit/7cc3aedb8d083a86de5cf4d92fd4c3190af4413d